.storybook/config.js

```diff
@@ -0,0 +1,12 @@
+import { configure } from '@storybook/react';
+import buildStory from './build-story';
+
+const requireStories = require.context('../', true, /components\/[^\/]+\/stories\/index.js$/);
```

Comment:
Would switching the subdirectories flag to `false` speed this up since this regexp is quite specific?

Reply:
Answer from IRL: "No because it won't ever traverse the directories"